cloud/azure/bin/setup-saml-keystore

```diff
@@ -0,0 +1,68 @@
+#! /usr/bin/env bash
+source bin/lib.sh
```

Comment:
suggestion (blocking): same issue with relative paths

the thing to do for this might be to export `PROJECT_BASE_DIR` early in command invocation so all scripts can assume it's available

Reply:
i'm realizing that suggestion is somewhat ambiguous with regard to this PR. for now maybe add TODOs for these and everywhere else we need to reference other files and create an issue for introducing a holistic approach.

Reply:
For now, I opted to define `PROJECT_BASE_DIR` in each file that's using it, with a TODO to figure out a better solution.